app/services/user.js

```diff
@@ -2,6 +2,12 @@ const { conflictError, databaseError } = require('../errors');
 const userModel = require('../models').user;
 const logger = require('.././logger');
 
+exports.findAllPagination = (limit, offset, orderField) =>
```

Comment:
You could receive an object here, passing too many parameters is tricky sometimes